### ISSUE TITLE
Bug: `prefer-math-min-max` reports false-positive for bigint

### ISSUE THREAD

**Issue by cruzdanilo** [sindresorhus/eslint-plugin-unicorn#2465]
the new rule `prefer-math-min-max` should ignore bigints

```js
const code = 420n > 69n ? 420n : 69n; // should be ok
```

**Comment by CamilleDrapier** [comment]
Just to add to this, Date should probably also be ignored by this rule (please elt me know if would be more appropriate to create a separate issue :bulb: ): 

```js
var date1 = new Date();
undefined
var date2 = new Date();
undefined
date1 < date2
true
date1 > date2 
false
```
